src/Deserializers/FingerprintDeserializer.php

```diff
@@ -16,10 +15,26 @@
  * @author Thomas Pellissier Tanon
  * @author Bene* < benestar.wikimedia@gmail.com >
  * @author Thiemo Mättig
+ * @author Adam Shorland
  */
 class FingerprintDeserializer implements Deserializer {
```

Comment:
I was also considering this when I went over it, though figured that it still provides sufficient value